src/ekore/anomalous_dimensions/unpolarized/time_like/as2.py

```diff
@@ -0,0 +1,741 @@
+"""The unpolarized time-like NLO Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+
+# from eko.constants import zeta2
+from numpy import power as npp
+
+from eko import constants
+
+from ....harmonics import w1, w2, w3
+from ....harmonics.constants import zeta2, zeta3
+from ....harmonics.polygamma import cern_polygamma as polygamma
+
+# from ....harmonics import cache as c
+
+
+@nb.njit(cache=True)
+def gamma_nsp(N, nf, cache=None, is_singlet=None):
```

Comment:
```suggestion
def gamma_nsp(N, nf, cache=None, is_singlet=False):
```

`is_singlet` has to be a boolean, you're also declaring as such in the docstring. Refrain from assign a value of a different type.